System.IO.Abstractions.TestingHelpers/MockFile.cs

```diff
@@ -354,7 +361,10 @@ public override void Move(string sourceFileName, string destFileName)
             {
                 throw CommonExceptions.FileNotFound(sourceFileName);
             }
-
+            if (!sourceFile.AllowedFileShare.HasFlag(FileShare.Delete))
+            {
+                throw new IOException("The process cannot access the file because it is being used by another process.");
```

Comment:
Please move this to `CommonExceptions`

Reply:
Moved this exception, and also the one that includes the file name of the file in use to the CommonExceptions class.